Tests/XCUITests/FxScreenGraph.swift

```diff
@@ -561,6 +561,7 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
         screenState.gesture(forAction: Action.ToggleNoImageMode) { userState in
             app.otherElements.tables.cells.switches[AccessibilityIdentifiers.Settings.BlockImages.title].tap()
         }
+        screenState.tap(app.buttons["Done"], to: HomePanelsScreen)
```

Comment:
Have you checked that other tests are not affected by this change? sometimes modifying the screengraph could affect other tests

Reply:
Yes, I have run all full functional tests on this change. I wondered why this line is needed, too.